### ISSUE TITLE
JobUploadListener::fileToString not needed

### ISSUE THREAD

**Issue by sergey-bleih** [gregurco/jobeet#7]
Hey, @hrapeichuk @gregurco I was covering `JobUploadListener` with unit tests and notices that the call to `JobUploadListener::fileToString` does not really seem necessary [here](https://github.com/gregurco/jobeet/blob/4e1ac000cca75ed432abd3e114bccc184f130b26/src/EventListener/JobUploadListener.php#L43) since `JobUploadListener::uploadFile`  will convert the file to string anyway. 
```php
private function uploadFile($entity)
{
    // upload only works for Job entities
    if (!$entity instanceof Job) {
        return;
    }
    $logoFile = $entity->getLogo();
    // only upload new files
    if ($logoFile instanceof UploadedFile) {
        $fileName = $this->uploader->upload($logoFile);
        $entity->setLogo($fileName);
    }
}
```

Let me know what you think, and I can adjust it with the test it self.

**Comment by gregurco** [comment]
Hey @sergey-bleih 
You are right. Method `uploadFile` populates logo in Job entity.

**Comment by gregurco** [comment]
Fixed in #8
